CarpetLib/src/restrict_3d_cc_o5_rf2.cc

```diff
@@ -167,8 +167,8 @@ void restrict_3d_cc_o5_rf2(
 
   if (not use_loopcontrol_in_operators) {
 
-// Loop over coarse region
-#pragma omp parallel for collapse(3)
+    // Loop over coarse region
+    // #pragma omp parallel for collapse(3)
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.